src/config/agoric/generateFromTemplate.spec.ts

```diff
@@ -0,0 +1,56 @@
+import { readFileSync } from "fs";
+import { createRequire } from "module";
+import { generateFromTemplate } from "./generateFromTemplate";
+
+const myRequire = createRequire(import.meta.url);
```

Comment:
Turadg asked why call it `myRequire`. I didn't have a good reason. I have started calling it `nodeRequre`, fwiw.

Reply:
Cool, `nodeRequire` works. I had considered `requireModule` but didn't like it

Reply:
[d7058da](https://github.com/DCFoundation/cosmos-proposal-builder/pull/27/commits/d7058dacdbb6b9af1fcc1300660839c306e4f936)